systemvm/debian/etc/sysctl.conf

```diff
@@ -60,3 +60,6 @@ net.ipv6.conf.all.forwarding = 0
 net.ipv6.conf.all.accept_ra = 0
 net.ipv6.conf.all.accept_redirects = 0
 net.ipv6.conf.all.autoconf = 0
+
+# Minimum swappiness without disabling it
+vm.swappiness=5
```

Comment:
Fine with me, 5% works, 1% works or even 0%.

If you would set it to 0% it would only swap in the worst-case scenarios, but this is fine as well.

Reply:
@wido thanks, 0% on newer kernels may disable swapping. I'm okay to go as low as 1% which means swapping is only used as a last resort. Will change the value to 1.

Reply:
Fixed to 1 now.